pkg/specgenutil/volumes.go

```diff
@@ -660,24 +659,16 @@ func getTmpfsMounts(tmpfsFlag []string) (map[string]spec.Mount, error) {
 }
 
 // validChownFlag ensures that the U or chown flag is correctly used
-func validChownFlag(flag string) (bool, error) {
-	kv := strings.SplitN(flag, "=", 2)
-	switch len(kv) {
-	case 1:
-	case 2:
-		// U=[true|false]
-		switch strings.ToLower(kv[1]) {
-		case "true":
-		case "false":
-			return false, nil
-		default:
-			return false, fmt.Errorf("'U' or 'chown' must be set to true or false, instead received %q: %w", kv[1], errOptionArg)
-		}
+func validChownFlag(value string) (bool, error) {
```

Comment:
Cleaned up this function quite a bit, going as far as not bothering to reparse `k=v` by changing function signature